bonus.md

```diff
@@ -3,11 +3,16 @@
 Edit this file to answer the following questions. Give examples. You may need to do a bit of research. :)
 
 1. Why is Hash.fetch useful?
+Hash.fetch is useful to get value if key exists otherwise returning a default value.
```

Comment:
Yep.  You could also do this with `@thing = args[:thing] || default_value` but `fetch` will work even if the value of `args[:thing]` is the boolean `false` whereas the pipe syntax will fail (if your default value is `true`, `@thing` will always be set to `true` even if you had `thing: false` in your hash).